web/tests/message_view.test.js

```diff
@@ -298,21 +315,8 @@ run_test("show_empty_narrow_message", ({mock_template}) => {
         ),
     );
 
-    // organization has disabled sending direct messages
-    realm.realm_private_message_policy =
```

Comment:
Some of these test changes need to be in the previous commit because the test fail on that commit.

Actually, you can also have the changes which changes the logic of checking permissions in this commit itself to avoid having intermidate changes in tests to remove the existing tests and then add this and a separate commit to remove the setting, both from the UI and backend.

Reply:
Removed `private_message_policy` from the backend and reordered the commits.